mysql-test/suite/galera/t/galera_insert_bulk.test

```diff
@@ -30,18 +36,28 @@ while ($count < 1000)
 
 --connection node_2
 
-# Disable bulk insert.
+# Disable bulk insert on this node.
 SET foreign_key_checks = 1;
 SET unique_checks = 1;
 
 # Insert a value out of the bulk insert range.
 INSERT INTO t1 VALUES (1001);
 
 --connection node_1
+
+# We need to trigger Galera-level certification conflict. For this:
+# - start applying single insert from the other node before bulk insert certifies
+# - certifying bulk insert will lead to the conflict
+# - keep applying single insert
+SET DEBUG_SYNC = 'wsrep_before_certification WAIT_FOR sync.wsrep_apply_cb_reached';
+SET DEBUG_SYNC = 'wsrep_after_certification SIGNAL signal.wsrep_apply_cb';
+
 --error ER_LOCK_DEADLOCK
 COMMIT;
 
```

Comment:
now jenkins test results show that the test fails here because COMMIT is successful, apparently certification failure does not happen after all

Reply:
jenkins test failure was due to using old galera library, the fix requires Galera 26.4.19